capstone/src/test/java/com/google/step/clients/RestaurantClientTest.java

```diff
@@ -78,7 +79,7 @@ public void correctEntityKind() {
     restaurantInfo.setProperty("score", 3.5);
     restaurantInfo.setProperty("status", "GOOD");
 
-    Restaurant actual = Restaurant.fromEntity(restaurantInfo);
+    Restaurant actual = RestaurantClient.fromEntity(restaurantInfo);
```

Comment:
Perhaps instead of testing the `fromEntity()` functionality since it should probably be private, you could test the public-facing that uses the `fromEntity()` functionality?

Reply:
I added tests for the three public-facing methods.